### ISSUE TITLE
[Feature Request]

### ISSUE THREAD

**Issue by drewburr** [sladkoff/minecraft-prometheus-exporter#260]
**Is your feature request related to a problem? Please describe.**

To help simplify the Prometheus configuration, it would be great if the `server_name` label populated by the exporter automatically. My thought is that if the provided dashboards can accept this information and, in a way, tend tend to expect it, it could make sense to have the exporter supply this value itself.

**Describe the solution you'd like**

The ability to tell the exporter what value should be set for the `server_name` label, and for it to have a sensible default (i.e. `minecraft`)

**Describe alternatives you've considered**

In my Kubernetes environment, it's inconvenient to update the Prometheus scrape configs to provide these values. Alternatively, I can have these labels added in using `relabelings` or `metricRelabelings` on the serviceMonitor instead, but this makes templating options like Helm a bit of a pain to manage. While setting up my serviceMonitors, I wondered if it made sense for the exporter to define this label.

**Additional context**

 Given that this label is a fully static value, the cost to storage should be negligible. It would also simplify setup for those who are less familiar with Prometheus and run multiple servers. In addition, I believe that the [suggested use of scrapeconfig](https://github.com/sladkoff/minecraft-prometheus-exporter/tree/master?tab=readme-ov-file#prometheus-config) would override any defaults that the exporter would set itself, meaning existing configuration shouldn't break.

**Comment by sladkoff** [comment]
@drewburr Makes sense to me. Would it be fine for you if the `server_name` can be set in the plugin config? 

As for the default, I'd suggest to do the following:

1. Use the Bukkit/Spigot API (if I remember correctly that this exists)
2. Fallback to static "minecraft" name

Do you agree?

**Comment by geethika870** [comment]
can i work on this issue!

**Comment by sladkoff** [comment]
@geethika870 sure, go for it!
